extras/indexing/src/test/java/mvm/rya/indexing/accumulo/temporal/AccumuloTemporalIndexerTest.java

```diff
@@ -237,17 +234,15 @@ public static void tearDownAfterClass() throws Exception {
     @Before
     public void setUp() throws Exception {
         conf = new Configuration();
+        conf.set(RdfCloudTripleStoreConfiguration.CONF_TBL_PREFIX, "triplestore_");
         conf.setBoolean(ConfigUtils.USE_MOCK_INSTANCE, true);
-        conf.set(ConfigUtils.TEMPORAL_TABLENAME, uniquePerTestTemporalIndexTableName);
         // This is from http://linkedevents.org/ontology
```

Comment:
The predicates used are from a published ontology.  I'll referenced it in the class javadoc comment.